Model/Payment/Method/Bread.php

```diff
@@ -463,6 +463,11 @@ protected function addTransactionInfo(
      */
     public function isAvailable(\Magento\Quote\Api\Data\CartInterface $quote = null)
     {
+
+        if(!$this->helper->aboveThreshold($quote->getGrandTotal())){
```

Comment:
this is still using the $quote variable before we're running the null check, so I think we should move the null check before this condition

Reply:
From what I checked it's never null, even on pdp checkout, don't think this will cause problem

Reply:
ok then in that case we should just remove the null default value in the function definition, and remove the null check all together. If we're going to have the null check present, then it needs to be the first thing, but if it's not needed, then it's just confusing to leave it in I think

Reply:
We can remove null check, but method needs to be compatible with parent class definition which has null default value

Reply:
ok well then in that case, I think it's just easier to leave the check in and move it up top. Since even if we don't think it'll ever be null, in the future it might be confusing why we have null as the default value, but never actually handle the null case.

Reply:
For isActive we need to return false in order to prevent method from being used, if that check is moved to top then it won't get to aboveThreshold check

Reply:
are you referring to the `!$quote->getBreadTransactionId()` part? if so we can just break out that condition into two parts, and have the `$quote === null` if statement first, then the aboveThreshold, then the getTransactionId if statement

Reply:
Ok you can add that, and then we run some checks, if it's ok I'm good with that.